### PR TITLE
feat: Cleanup values.yaml with sensible defaults

### PR DESCRIPTION
**Pull request by obs-gh-enricogiorio**
Send everything to the /v1/kubernetes/v1/entity path

### CODE REVIEW

**Comment by obs-gh-nikhildua** [comment]
https://github.com/observeinc/helm-charts/tree/main/integration/modules/deploy_helm/values
These are the values file that the integration tests refers to

Entity token is not specified for testing so it'll refer to whatever is in the default values file. Do we now need one?

**Comment by obs-gh-enricogiorio** [comment]
I don't think we need one, I need to look into this very carefully. Will get back to you tomorrow

**Comment by obs-gh-enricogiorio** [comment]
I see that the values are parametric themselves, i.e. they refer to stuff like {$observe_token}. Where are these variables defined?
It's definitely a test configuration issue since the patch changes the defaults and we might need to change the overrides as well.

File: charts/agent/values.yaml [hunk 1]
```diff
@@ -152,9 +150,7 @@ cluster-events:
 
   image:
     repository: observeinc/observe-agent
-    # Overrides the image tag whose default is the chart appVersion.
-    # The service's name will be applied to the end of this value.
-    tag: "1.3.0"
+    tag: latest
```
Comment (xueweiz) [inline charts/agent/values.yaml:153]: Hum... Not pinning the image is likely too dangerous to be used as default. As user could see their agent "secretively" updating. Why do we need this as default?

**Review by xueweiz — APPROVED** [review]
LGTM! But you need to regenerate some files to pass the presubmit checks I think

**Review by xueweiz — APPROVED** [review]

**Pull request merged**